### DISCUSSION TITLE
右键任务栏会重启资源管理器

### DISCUSSION THREAD

**Discussion by FKcrown** [zetaloop/ExplorerPatcher#21] [General]
Windows11 升级了`23403.1001`后，只要右键任务栏资源管理器就会重启，已安装最新版本ExplorerPatcher（22621.1344.53.1），不知道是新版本不兼容还是和软件某个设置冲突了

**Comment by zetaloop** [comment]
应该就是ep的问题，某些版本又不兼容了。等待作者更新适配，如果影响使用的话，可以试试换个旧版。另外，一部分功能即使卸载软件依然会保持的，比如右键菜单的修改等。

**Reply by zetaloop** [reply]
我这儿（Win11 Beta 22631.2048）现在使用 Win11 任务栏右键会闪退，但是切换为 Win10 任务栏就可以正常使用。

**Reply by FKcrown** [reply]
进入dev频道后没用过ep了，现在版本号是`23506.1000`，在`23481`版本的时候TranslucentTB也不能兼容新版任务栏了，只能暂时放弃任务栏美化了
![image](https://github.com/zetaloop/ExplorerPatcher/assets/79238509/27b3eb9b-65bb-48f2-b164-fdda6900eabf)

**Comment by KIITTID** [comment]
昨天更新这一版会导致右键重启资源管理器

**Reply by zetaloop** [reply]
最新的内测版是我自己测试用，汉化尚未完成，请暂时不要使用。
如果你是使用官方版也会出现右键重启，请把相关问题反馈给原作者 valinet。

**Reply by KIITTID** [reply]
应该是新系统版本22621.2070的问题，手贱不该升级啊

**Reply by zetaloop** [reply]
在高级里打开“不修改任务栏右键菜单”，右键任务栏就可以恢复正常

**Reply by KIITTID** [reply]
🆗
